runtime/src/main/java/org/corfudb/runtime/clients/ManagementClient.java

```diff
@@ -132,4 +133,12 @@ public QueryResponse queryRequest(UUID workflowId) {
                 .payloadMsg(req));
         return (QueryResponse) CFUtils.getUninterruptibly(resp).getResponse();
     }
+
```

Comment:
Ideally, there should be comments (just one line) telling what each of these requests do.

Reply:
done.